lib/plugins/rawbg.js

```diff
@@ -49,14 +56,18 @@ function init (ctx) {
     sbx.pluginBase.updatePillText(rawbg, options);
   };
 
-  rawbg.calc = function calc(sgv, cal) {
+  rawbg.calc = function calc(sgv, cal, sbx) {
     var raw = 0;
     var cleaned = cleanValues(sgv, cal);
 
+    var prefs = rawbg.getPrefs(sbx);
+
     if (cleaned.slope === 0 || cleaned.unfiltered === 0 || cleaned.scale === 0) {
       raw = 0;
-    } else if (cleaned.filtered === 0 || sgv.mgdl < 40) {
+    } else if (cleaned.filtered === 0 || sgv.mgdl < 40 || prefs.unfiltered) {
       raw = cleaned.scale * (cleaned.unfiltered - cleaned.intercept) / cleaned.slope;
+    } else if (prefs.smooth) {
```

Comment:
This part of the code will never be reached if `unfiltered==true`.  Is this inteded?

Reply:
Yes, the smooth preference allows the user to select how the raw `filtered` data is displayed, smooth or exaggerated.

If the user wants either of those plotted (both have some type of filtering applied), that would be opposite of the user wanting the raw `unfiltered` values plotted.